packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/proposal/acceptProposal.ts

```diff
@@ -24,7 +24,17 @@ const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInpu
   if (flags.input) return flags.input as CommandInput
   const { rdd: rddPath, secret } = flags
 
-  if (!rddPath) throw new Error('RDD flag is required. Provide it with --rdd flag')
+  if (!rddPath) {
+    throw new Error('RDD flag is required. Provide it with --rdd flag')
+  }
```

Comment:
This will now be hidden behind `RDD.getRDD` call once this PR is merged: https://github.com/smartcontractkit/chainlink-terra/pull/195

cc @reductionista